packages/formation-react/src/components/ErrorableCheckboxGroup/ErrorableCheckboxGroup.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ErrorableCheckboxGroup from './ErrorableCheckboxGroup';
+
+export default {
+  title: 'Library/ErrorableCheckboxGroup',
+  component: ErrorableCheckboxGroup,
+  argTypes: {
+    label: { control: 'text' },
+    id: { control: 'text' },
+    errorMessage: { control: 'text' },
+    label: { control: 'text' },
+    required: { control: 'boolean' },
+    options: {
+      control: 'object',
+    },
+    values: {
+      control: 'object',
+    },
+  },
```

Comment:
Same story as the other PRs, I don't think these are needed because the `propTypes` are defined on the component.

Reply:
So I was able to remove all `argTypes` except for `options`. Setting the control prop to "object" i'm able to update the content using controls but if i remove it im unable to update the content